packages/entity-browser/src/util/api/entities.js

```diff
@@ -87,10 +92,11 @@ export const entitiesListTransformer = json => {
 const defaultEntitiesTransformer = json => (json)
 
 export function fetchEntities(entityName, page, orderBy = {}, limit, fields = [],
-                              searchInputs = {}, transformer = defaultEntitiesTransformer) {
+                              searchInputs = {}, transformer = defaultEntitiesTransformer, formName) {
```

Comment:
i would move formName before transformer to make them at least a bit ordered. maybe consider a default value to make it clear by the look at the signature that param is not mandatory?